### ISSUE TITLE
Multiline output

### ISSUE THREAD

**Issue by tauboy** [auxesis/cucumber-nagios#8]
Hi auxesis

I was wondering if you had any plans to extend the output returned via `cucumber-nagios`? As Nagios allows multiline output from plugins, I was thinking it would good to cat the type of response you get when you run a Cucumber test against a feature or group of features, e.g. instead of:-

```
 Critical: 0, Warning: 0, 4 okay | passed=4, failed=0, nosteps=0, total=4
```

it could return:-

```
Critical: 0, Warning: 0, 4 okay\n 2 scenarios (2 passed)\n 4 steps (4 passed) | passed=4, failed=0, nosteps=0, total=4
```

Then the main view in Nagios would display the first line of the response (Critical: 0, Warning: 0, 4 okay) but if you clicked on the check you would be able to see the extra data: 

```
Critical: 0, Warning: 0, 4 okay
2 scenarios (2 passed)
4 steps (4 passed)
```

Cheers,
Mike

**Comment by auxesis** [comment]
Hi Mike,
Totally possible. I'll add it to the TODO.

**Comment by tauboy** [comment]
opps just reread the api doc and the output for multiline is

TEXT OUTPUT | OPTIONAL PERFDATA
LONG TEXT LINE 1
LONG TEXT LINE 2
...
LONG TEXT LINE N | PERFDATA LINE 2
PERFDATA LINE 3
...
PERFDATA LINE N

**Comment by auxesis** [comment]
Hi Mike,
This has been implemented as part of cucumber-nagios 0.8.8!

Closing, but let me know if you discover any bugs!

**Comment by tauboy** [comment]
Will test this today and let you know if I find anything
